types/jasmine/jasmine-tests.ts

```diff
@@ -1053,20 +1053,42 @@ describe("jasmine.objectContaining", () => {
         };
     });
 
-    it("matches objects with the expect key/value pairs", () => {
-        // not explictly providing the type on objectContaining only guards against
-        // missmatching types on know properties
+    it("matches objects with the correct type for known properties", () => {
+        // not explicitly providing the type on objectContaining only guards against
+        // mismatching types on known properties
+
+        // this does not cause an error as the compiler cannot infer the type completely
+        expect(foo).not.toEqual(jasmine.objectContaining({
+            a: 37,
+            foo: 2,
+        }));
+
+        // Contrary to the test in ../jasmine-tests.ts, this does not cause an error
```

Comment:
This is a bad copypasta comment.
../jasmine-tests.ts doesn't exist

Try

Contrary to the test in the ts3.1/ directory, ...